src/foam/graphics/ScrollCView.js

```diff
@@ -109,12 +109,14 @@ foam.CLASS({
       expression: function(size, extent, yMax, innerBorder) {
         return size ? ( yMax - innerBorder ) / (size - extent) : 0;
       }
-    }
+    },
+    'mouseEvent'
   ],
 
   methods: [
     function initCView() {
       this.canvas.pointer.touch.sub(this.onTouch);
+      this.mouseEvent$.sub(this.updateScrollFromEvent);
```

Comment:
Might want to do:
this.onDetach(this.mouseEvent$.sub(this.updateScrollFromEvent))